File: Silver/Cooper/Android/FirstAndroidApp/MainActivity.swift
import java.util
import android.app
import android.content
import android.os
import android.util
import android.view
import android.widget

public class MainActivity: Activity {

	static let EXTRA_MESSAGE = "com.example.firstandroidapp.MESSAGE";

	public override func onCreate(savedInstanceState: Bundle!) {
		super.onCreate(savedInstanceState);

		// Set our view from the "main" layout resource
		ContentView = R.layout.main;
	}

	public func sendMessage(view: View) {

		let intent = android.content.Intent(self, typeOf(DisplayMessageActivity));
		let editText = findViewById(R.id.edit_message) as! EditText;
		let message = editText.getText().toString();
		intent.putExtra(EXTRA_MESSAGE, message);
		startActivity(intent);
	}
}

File: Iodine/Cooper/Android/FirstAndroidApp/MainActivity.java
package org.me.firstandroidapp;

import java.util.*;
import android.app.*;
import android.content.*;
import android.os.*;
import android.util.*;
import android.view.*;
import android.widget.*;

public class MainActivity extends Activity
{

	public static final String EXTRA_MESSAGE = "com.example.firstandroidapp.MESSAGE";

	@Override
	public void onCreate(Bundle savedInstanceState)
	{
		super.onCreate(savedInstanceState);

		// Set our view from the "main" layout resource
		ContentView = R.layout.main;
	}

	public void sendMessage(View view)
	{
	   Intent intent = new Intent(this, typeOf(DisplayMessageActivity));
	   EditText editText = (EditText)findViewById(R.id.edit_message);
	   String message = editText.getText().toString();
	   intent.putExtra(EXTRA_MESSAGE, message);
	   startActivity(intent);
	}
}